scripts/youtube.py

```diff
@@ -0,0 +1,20 @@
+import webbrowser
+import requests
+from bs4 import BeautifulSoup
+
+query=input('Enter the song to be played: ')
+query=query.replace(' ','+')
```

Comment:
instead of taking input here, i guess it's better to provide a command line option like `script_name -s song_name`?

Reply:
ok now you can do 
:$scriptname songname